text/us/courses.h

```diff
@@ -1,80 +1,80 @@
-COURSE_ACTS(COURSE_BOB, _(" 1 BOB-OMB BATTLEFIELD"),
```

Comment:
TODO: Add `_(` `)` shells back to text strings, instead using a macro to make `_("string")` evalutate to `"string"`, to avoid merge conflict issues

Reply:
Decided not worth it - merge conflicts within the text files are already going to have many necessary merge conflicts as is, due to characters being changed